<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-SI8EPIA" name="com.assertion.AssertionTest" tests="1" failures="1" timestamp="19 Dec 2020 06:06:30 GMT" time="0.068" errors="0">
  <testcase name="SoftAssertTest" time="0.068" classname="com.assertion.AssertionTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Matching title expected [Review your itinerary] but found [[[ChromeDriver: chrome on WINDOWS (8877c2eec58b612bdf9f885936e77956)] -&gt; xpath: //div/h2[.=&#039;Review your itinerary&#039;]]]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Matching title expected [Review your itinerary] but found [[[ChromeDriver: chrome on WINDOWS (8877c2eec58b612bdf9f885936e77956)] -> xpath: //div/h2[.='Review your itinerary']]]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at com.assertion.AssertionTest.SoftAssertTest(AssertionTest.java:49)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- SoftAssertTest -->
</testsuite> <!-- com.assertion.AssertionTest -->
